masq_lib/src/shared_schema.rs

```diff
@@ -48,6 +48,18 @@ pub const EARNING_WALLET_HELP: &str =
      (case-insensitive). If you already have a derivation-path earning wallet, don't supply this. \
      If you have supplied an earning wallet address before, either don't supply it again or be \
      careful to supply exactly the same one you supplied before.";
+pub const EXIT_LOCATION_HELP: &str =
+    "Choose your Exit Location for access the internet. You can choose from all countries, available in \
+    your Neighborhood. \n\
+    You can set exit-location --country-codes \"CZ\" without --fallback-routing argument and in that case, \
+    the fallback-routing will be set OFF. In case, you do not specify the --country-codes and neither --fallback-routing, \
+    then country-codes will be set OFF and fallback-routing will be set ON.\n\n\
```

Comment:
If you activate exit-location preferences, all exit Nodes in countries you don't specify will be prohibited: that is, if there is no exit Node available in any of your preferred countries, you'll get an error. However, if you just want to make a suggestion, and you don't mind Nodes in other countries being used if nothing is available in your preferred countries, you can specify --fallback-routing, and you'll get no error unless there are no exit Nodes available anywhere.

Here are some example commands: